keras_trainer/trainer.py

```diff
@@ -266,15 +327,21 @@ def run(self):
             loss_weights=self.loss_weights
         )
 
+        if self.train_generator.samples // self.batch_size == 0:
+            raise ValueError('Batch size is higher than the total number of training samples')
+
+        if self.val_generator.samples // self.batch_size == 0:
+            raise ValueError('Batch size is higher than the total number of validation samples')
```

Comment:
I don't think the Trainer class and run method works for no validation set at the moment. If we wanted to do that maybe it would be a good addition. I added the check because I was getting errors and I believe now it is easier to debug.

Reply:
cool ya makes sense! Looks good